### PR TITLE
Update main README and demo/README

### PR DESCRIPTION
**Pull request by dwalton76**

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -1,13 +1,44 @@
...
+To install these on your EV3, use git to clone the ev3dev-lang-python repository
+from github. Your EV3 will need Internet connectivty in order to clone the
+repository from github.
+```
+$ sudo apt-get install git
+$ git clone https://github.com/rhempel/ev3dev-lang-python.git
+```
+
+## Running A Program
+There are two ways to run a program.  You can run a program from the command line
```
Comment (WasabiFan) [inline demo/README.md]: I think these are double spaces; can we replace them with singles?

File: demo/README.md [hunk 2]
```diff
@@ -1,13 +1,44 @@
...
+or from the brickman interface.  Note that for both running from the command line
+and running from Brickman the program must be marked as an executable and the
```
Comment (WasabiFan) [inline demo/README.md]: Can we separate the most important sentence(s) into a new paragraph and bold them?

File: demo/README.md [hunk 3]
```diff
@@ -1,13 +1,44 @@
...
+```
+## Brickman
+To run one of the demo programs from Brickman, select the program in the
+File Browser.
+
+## Demo Programs
 ### BALANC3R
 
-Laurens Valk's BALANC3R - This robot uses the gyro sensor to balance on two wheels. Use the IR remote to control BALANC3R
+Laurens Valk's BALANC3R - This robot uses the gyro sensor to balance on two
```
Comment (WasabiFan) [inline demo/README.md]: Nice catch in the wrapping!

File: README.rst [hunk 4]
```diff
@@ -126,6 +126,9 @@ for ev3dev. The first allows you to run this program from Brickman, while the
 second imports this library.
 
 When saving Python files, it is best to use the ``.py`` extension, e.g. ``my-file.py``.
+To be able to run your Python code, your program must be executable. To mark a
```
Comment (WasabiFan) [inline README.rst]: Can we bold the important bits?

File: demo/README.md (hunk 2 above)
Comment (dwalton76) [inline demo/README.md]: will do

**Pull request merged**